### ISSUE TITLE
Is possible specify a scenario when using @apiResource?

### ISSUE THREAD

**Issue by netojose** [knuckleswtf/scribe#488]
- [x] I've read the [documentation](https://scribe.knuckles.wtf/laravel) and I can't find details on how to achieve this.

I tried to specify a scenario label, but didn't work:
```php
@apiResource scenario="View my data" App\Http\Resources\UserMe // causes an error
@apiResource App\Http\Resources\UserMe scenario="View my data" // causes an error
```

Since I can't set a scenario, on POSTMAN the text is "Untitled Example"
<img width="302" alt="Screen Shot 2022-06-26 at 11 20 59" src="https://user-images.githubusercontent.com/134008/175809729-2571c8dc-9f9e-44eb-a675-700d8cd39358.png">

**Comment by shalvah** [comment]
Ah, seems like an oversight.

**Comment by shalvah** [comment]
Released in 3.33.0.

**Comment by netojose** [comment]
Awesome, everything works as expected. Thanks <3

<img width="277" alt="Screen Shot 2022-06-28 at 01 05 02" src="https://user-images.githubusercontent.com/134008/176059490-97f3d6fc-65ae-47a4-a3b0-94247e5691d9.png">
